recipes/mariadb-connector-c/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e -x
+
+mkdir build
+cd build
+
+cmake ${CMAKE_ARGS} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+    -DCMAKE_SKIP_INSTALL_ALL_DEPENDENCY=ON \
+    ..
+
+make -k -j${CPU_COUNT}
+
```

Comment:
Same question as above: Is there a make check or ctest that we can run here?

Reply:
I added ctest to build.sh and build.bat